8.x/getting-started/installation.md

```diff
@@ -269,152 +557,322 @@ Run the following command to install Composer in the /usr/local/bin directory:
+To hook direnv into your shell, add the following line to the end of `~ /.bashrc` file by these steps:
+1. Open the `/.bashrc` file.
 
 ```bash
-cd template-project
-npm install
+cd path/to/larammerce_base_theme
+vim ~/.bashrc
 ```
 
-Then to build the project resource files run:
+2. Enable editing with `shift+g+o+enter`.
+3. Add the line below at the end of the file:
 ```bash
-npm run prod # to build and minify the project resource files.
-npm run watch # to watch for resource file changes and then build and export them after every change.
+eval "$(direnv hook bash)"
+``` 
+- Open `.envrc` and add this line on top of the file:
+```bash
+export ECOMMERCE_BASE_PATH=/home/your-sysytem-name/your-root/larammerce
 ```
+Note: On the main directory run `pwd` to get the path used in the command above.
 
-Create a new .envrc file from the .envrc.example file
-
+Here, you may run to an access issue, to solve it run:
 ```bash
-cp .envrc.example .envrc
+direnv allow . 
 ```
-Add the following to the file named `.envrc`:
 
-```bash
-export ECOMMERCE_BASE_PATH=/var/www/larammerce 
+To test the accuracy of this command, run:
+
+``` bash
+echo  $ECOMMERCE_BASE_PATH
 ```
+The result should be the path you equalized before.
+
+Run the following command to install node requirements:
 
-Then this command to make the direnv package know this directory as its list of directories:
+```bash
+cd path/to/the/larammerce/project
+npm install
+```
+Then run this command to build the resource files:
 
 ```bash
-direnv allow .
+npm run prod 
 ```
-And after each change in the template, we enter this command again to deploy the resources to the backend directory:
+
+
+ Everything is ready to deploy the project on `localhost:8080`. Execute:
+ ```bash
+ php -S 0.0.0.0:8080 -t public_html/
+ ```
+
+ To enable a basic theme on the project, run this command on ` ~/.../larammerce-base-theme` :
 
 ```bash
 ./deploy.sh
 ```
 
+---
+
+#### Working with the theme
+
+As a sample, lets build a web page! 
+
+Once you logged in as admin on `localhost:8080/admin`, press plus button on the left buttom of the page ,create a folder and equalized some random values in the form displayed. 
+
+For example:
+
+ | Field name | Value |
+ |------------|-------|
+ |Title: | Test folder|
+ |url part: | test |
+ |Preference : | 0|
+ |Content type :| regular|
+ |Enabling web page| on |
+
+ Save changes!
+
+Afterward, another form will be displayed. Here you may need to build a theme file prior to editing the web page.
+To do so, follow this step-by-step instruction:
+
+1. Create a file on `/larammerce-base-theme/public/views`.
+
+```
+ touch test.blade.php
+```
+2.Add the script below to the file:
+
+```
+@php
+
+echo "hello world!"
+
+@endphp
+```
+3.Now deploy the project on `~ /larammerce-base-theme` directory:
+```
+./deploy.sh 
+```
+
+Finally, press `edit web page` button in the form, select the url name you created, and save.(here the url name is test)<br> 
```

Comment:
select the url name -> select the blade file name

here the url name is test -> Here the blade file name is `test`